### ISSUE TITLE
Accessibility: Some form elements missing labels

### ISSUE THREAD

**Issue by bedmison** [web-cat/code-workout#239]
Form elements need labels so screen readers can properly announce the associated form field. For example, this shows up in the listings of exercises 

Shows up in the CodeMirror interface.

----

From the WCAG 2.1 spec:

To solve this problem, you need to fix at least (1) of the following:

- Element does not have an implicit (wrapped) <label>
- Element does not have an explicit <label>
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute
- Element has no placeholder attribute
- Element's default semantics were not overridden with role="none" or role="presentation"

---
Solution? : Update code mirror and retest.
---

**Comment by Koenisegg484** [comment]
Hi, I would love to work on this issue
